hack/check-docs.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-readonly DOCS_CHANGES=`git diff --name-status main | grep "docs/" | wc -l`
+readonly DOCS_CHANGES=`git diff --name-status main | grep "docs-v1/" | wc -l`
```

Comment:
This actually need to check both v1 and v2, and test building the corresponding doc site. As we will have another pr to do the renaming. I'm OK to leave it as it is for now.

Reply:
done